hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricDescriptorConstants.java

```diff
@@ -574,6 +574,10 @@ public final class MetricDescriptorConstants {
     // ===[/THREAD]=====================================================
 
     // ===[TSTORE]======================================================
+    public static final String DEVICE_DISCRIMINATOR_NAME = "name";
```

Comment:
Consider prefixing with `TSTORE_` this as well.

Reply:
https://github.com/hazelcast/hazelcast/pull/21911/commits/7be9f60575f2fc69600c606eefa20ae925135df7
https://github.com/hazelcast/hazelcast-enterprise/pull/5178/commits/e4cfedcf84c1870fb47b6c5ba3b1509978c2c210